cabal-testsuite/PackageTests/ShowBuildInfo/A/build-info-unknown.out

```diff
@@ -0,0 +1,12 @@
+# cabal show-build-info
+cabal: Internal error in target matching. It should always be possible to find a syntax that's sufficiently qualified to give an unambiguous match. However when matching 'exe:B'  we found exe:B (unknown-component) which does not have an unambiguous syntax. The possible syntax and the targets they match are as follows:
```

Comment:
Um, this sounds like a cabal bug to me. Certainly nothing that should be expected test output or ever seen by users! Please find out what's going on here.

Reply:
That error message looks like #5081.